rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -230,34 +233,21 @@ pub async fn main(mut cli: Cli) -> Result<(), Error> {
     let (channel_snapshot_send, channel_snapshot_recv) = tokio::sync::watch::channel(None);
 
     // Registry Client
-    let registry_client = if let Some(v) = &cli.registry.registry_local_store_path {
-        // Notice no-op logger
-        let logger = ic_logger::new_replica_logger(
-            slog::Logger::root(tracing_slog::TracingSlogDrain, slog::o!()),
-            &ic_config::logger::Config::default(),
-        );
-
-        let nns_pub_key = cli
-            .registry
-            .registry_nns_pub_key_pem
-            .as_ref()
-            .map(|path| parse_threshold_sig_key(path).expect("failed to parse NNS public key"));
-
-        let registry_replicator = RegistryReplicator::new(
-            logger,
-            v,
-            cli.registry.registry_nns_poll_interval,
-            cli.registry.registry_nns_urls.clone(),
-            nns_pub_key,
-        )
-        .await;
+    let registry_client = if let Some(local_store_path) = &cli.registry.registry_local_store_path {
```

Comment:
```suggestion
    let registry_client = if let Some(v) = &cli.registry.registry_local_store_path {
```

Ah, and I did not feel like this `Some(v)` was super descriptive, thus the rename. I thought this was a one-off, but now I see that you have the same style everywhere below, so feel free to revert this rename.